src/platform/tls_openssl.c

```diff
@@ -841,6 +852,71 @@ CxPlatTlsSecConfigCreate(
                 Status = QUIC_STATUS_TLS_ERROR;
                 goto Exit;
             }
+        } else if (CredConfig->Type == QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12) {
+            BIO* bio = BIO_new(BIO_s_mem());
+            PKCS12 *p12 = NULL;
+
+            if (bio) {
```

Comment:
Is it allowed for `bio` to be `NULL`? That would seem to be an error condition, right? I'd rather check for `!bio` and log that error and bail, instead of the way you're currently doing it.

Reply:
yes, BIO_new() can return null on failure. AFAIK this would happen only if you run out of memory but I can certainly unwind the error check. Since that unlikely IMHO I lump it with check for valid PKCS12 object.

Reply:
moved error check for bio to separate block.